### ISSUE TITLE
Does not work, desktop mode does nothing

### ISSUE THREAD

**Issue by abadoxx** [matzman666/OpenVR-InputEmulator#148]
I have tried everything to get this program to work and everytime I try to click the app from within steamVR nothing pops up. Desktop mode does nothing. Is there another way to get keybinds to work outside of steam vr?

**Comment by abadoxx** [comment]
Well I finally got it working by patching the drivers and now when I goto input mapping it is just "analog mapping" no digital mapping. It seems to give me digital options in desktop mode but not inside VR, odd.

Well even changing the inputs to keyboard commands doesn't actually do anything, I officially give up

**Comment by UeflaVR** [comment]
Same here. Maybe if we got an old version of steamvr we solve that. I deleted an old version and now nothing to do...by now...:/

**Comment by marcob2178** [comment]
Hi. Is there anyone that found a working solution ?

**Comment by Warpup** [comment]
Not working on my end, just installed it and crashes to safemode

**Comment by trueA1M2** [comment]
Have a read of this dude 
https://github.com/matzman666/OpenVR-InputEmulator/issues/143

**Comment by WiFiCo** [comment]
@marcob2178 no, I've spent 2 weeks off and on trying to figure out a fix for setting a digital input (keypress) remapped to the menu button to no avail. The "fix" can help keep it from crashing outright, but it doesn't fix the functionality. Hopefully we get a fix one day.

**Comment by UeflaVR** [comment]
I'm using again openvr Input emulator (inside hmd) . Just backing to an old version of steamvr I found. Also with that Revive is flawless again.

**Comment by marcob2178** [comment]
> @marcob2178 no, I've spent 2 weeks off and on trying to figure out a fix for setting a digital input (keypress) remapped to the menu button to no avail. The "fix" can help keep it from crashing outright, but it doesn't fix the functionality. Hopefully we get a fix one day.

Thanks for the reply. I hope you will find 1 day the fix for the problem valve created

**Comment by marcob2178** [comment]
> I'm using again openvr Input emulator (inside hmd) . Just backing to an old version of steamvr I found. Also with that Revive is flawless again.

Could you please let me know which old version of  steamvr you are using that I should download please ?
